### DISCUSSION TITLE
reco_arch='sar_resnet31'

### DISCUSSION THREAD

**Discussion by FarnazZeidi** [mindee/doctr#1011] [Q&A]
Hello,
Festival thank you so much for your amazing models.

I have problem with **ocr_predictor** when i want to change 'crnn_vgg16_bn' to 'sar_resnet31':

  - #model = ocr_predictor(det_arch='db_resnet50', reco_arch='crnn_vgg16_bn', pretrained=True)
  - model = ocr_predictor(det_arch='db_resnet50', reco_arch='sar_resnet31', pretrained=True)
 

I received this WARNING:
  
  - WARNING:root:Invalid model URL, using default initialization

But there is no such this WARNING if i want to use this model in  **recognition_predictor** :

  - predictor = recognition_predictor('sar_resnet31')

Why I have this WARNING with  ocr_predictor? could you please help me? 
Thank you so much in advance

**Comment by frgfm** [comment]
Hello @FarnazZeidi :wave:

First, if you encounter a problem, it's better to open a GH issue than a discussion :) There is a template so that you provide all the relevant information to debug.

Now, back to your problem, I see no reason why there would be a difference between the two. Can you confirm that you're using the same deep learning backend between the two snippets? :thinking: 
Which version of docTR are you using?

(I suggest opening a GH issue [here](https://github.com/mindee/doctr/issues/new?assignees=&labels=type%3A+bug&template=bug_report.yml), it will guide you to collect this information :))

Cheers :v:

**Reply by felixdittrich92** [reply]
Version 0.5.1 (current pip) (pytorch) sar has no pretrained checkpoint model will be initialized random
Version 0.6.0 (main) (tensorflow and pytorch) master and sar fixed model has in both frameworks no pretrained checkpoint
